src/tool_help.c

```diff
@@ -68,7 +69,7 @@ static const struct helptxt helptext[] = {
   {"    --cert-status",
    "Verify the status of the server certificate"},
   {"    --cert-type <type>",
-   "Certificate file type (DER/PEM/ENG)"},
+   "Certificate file type"},
```

Comment:
The line does not seem to be too long, maybe it is worth keeping the few available options?

Reply:
I didn't intentionally change this, as the commit message says. Someone must've edited the generated source previously and not the correct source file...